src/test/kotlin/com/exactpro/th2/codec/oracle/logminer/LogMinerTransformerTest.kt

```diff
@@ -265,4 +283,50 @@ class LogMinerTransformerTest {
             }
         }
     }
+
+    companion object {
+        private val UNEXPECTED_TOKEN_TEXT = """
+                        net.sf.jsqlparser.parser.ParseException: Encountered unexpected token: "broken" <S_IDENTIFIER>
```

Comment:
Do we need the whole error message here? Maybe we can check the beginning of the error message?..